internal/handlers/vault_handler.go

```diff
@@ -117,6 +117,7 @@ func (a *Api) getVaultByUIDHandler(c *gin.Context) {
 	vaultResp := models.VaultResponse{
 		UId:            vault.Uid,
 		Name:           vault.Alias,
```

Comment:
can we also set `Name` appropriately? like Name: vault.Name

Reply:
Do we want to share the vault name with others?
I assumed that's something private that the user might not want to share.
Actually, when the user renames the vault, it changes the vault alias, not the vault name (the vault name comes from the QR code).
Maybe I can set  it to empty string

Reply:
oh ,ok